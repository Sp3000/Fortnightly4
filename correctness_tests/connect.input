place 0 0 red
count
place 0 1 red
count
connect 0 0 0 1
count
place 0 2 blue
count
connect 0 0 0 2
count
connect 0 1 0 2
count