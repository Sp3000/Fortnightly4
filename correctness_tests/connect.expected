1
2
1
2
Error: connect 0 0 0 2
Error: connect 1 1 1 2
2
1
